src/telemetry/backend-metrics.yaml

```diff
@@ -0,0 +1,338 @@
+        type: string
+      utm_medium:
+        description: Attribution medium.
+        type: string
+      utm_source:
+        description: Attribution source.
+        type: string
+      utm_term:
+        description: Attribution term.
+        type: string
+      referrer:
+        description: The referring URL, if any.
+        type: string
+
+subscription:
+  user_id:
+    description: Mozilla Accounts user ID.
+    type: string
+    lifetime: application
+    send_in_pings:
+      - events
+    notification_emails:
+      - rhelmer@mozilla.com
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    expires: never
+    data_sensitivity:
+      - technical
+
+  session_id:
+    description: An ID that allows us to track “sessions” of the user experience within the product.
+    type: string
+    lifetime: application
+    send_in_pings:
+      - events
+    notification_emails:
+      - rhelmer@mozilla.com
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    expires: never
+    data_sensitivity:
+      - technical
+
+  monitor_user_id:
+    description: Mozilla Monitor-specific user ID.
+    type: string
+    lifetime: application
+    send_in_pings:
+      - events
+    notification_emails:
+      - rhelmer@mozilla.com
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    expires: never
+    data_sensitivity:
+      - technical
+
+  plan_tier:
+    description: Which tier of plan the user is on [Free, Plus]
+    type: string
+    lifetime: application
+    send_in_pings:
+      - events
+    notification_emails:
+      - rhelmer@mozilla.com
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    expires: never
+    data_sensitivity:
+      - technical
+
+  activate:
+    type: event
+    description: |
+      A subscription has been activated for this user.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - rhelmer@mozilla.com
+    expires: never
+    send_in_pings:
+      - technical
+
+  cancel:
+    type: event
+    description: |
+      A subscription cancellation has taken effect, and the user is no longer subscribed to Monitor Plus.
+    bugs:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1896222
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - rhelmer@mozilla.com
+    expires: never
+    send_in_pings:
+      - technical
+
```

Comment:
Additional event we might want to track is applying coupon code to a subscription